baseline/lexicon.yaml

```
@@ -163,6 +180,24 @@
     multiple forms of identification.
   synonyms:
     - MFA
+- term: Openchain
```

I don't want to nitpick but i think this is how they spell it :/
```suggestion
- term: OpenChain
```